properties/color/ui.json

```diff
@@ -5,6 +5,12 @@
 			"border": {
 				"value": [46, 62, 72, 0.12]
 			},
+			"borderCard": {
```

Comment:
If these colors have other uses (for example, avatar backgrounds), we should give it a more neutral name if possible